sentry-core/src/main/java/io/sentry/core/hints/Resettable.java

```diff
@@ -0,0 +1,7 @@
+package io.sentry.core.hints;
+
+/** marker interface that resets/restarts a Flushable operation. */
+public interface Resettable {
```

Comment:
Does this need to be public? It's not really `reset a flushable operation` as it's reset fields of the hint so I can use the instance again. It was meant only to be used in a private class as an optimization to no `new` up a new instance for each file.

Perhaps better to delete this hint and create a new hint then to reduce complexity.

Reply:
yeah bad written, my bad.
all the hints are under the `hints` package and as you know, java is like this.

Reply:
Yeah makes sense. I guess we can manage to get rid of this one then at least